crates/biome_css_parser/Cargo.toml

```diff
@@ -22,11 +21,11 @@ biome_rowan            = { workspace = true }
 tracing                = { workspace = true }
 
 [dev-dependencies]
-biome_test_utils  = { workspace = true }
+biome_test_utils  = { path = "../biome_test_utils" }
```

Comment:
Use the path dependency to avoid requiring these crates to be published when doing cargo publish